libs/sr/ui/src/LightCone/LightConeCard.tsx

```diff
@@ -0,0 +1,28 @@
+import { CardThemed } from '@genshin-optimizer/common/ui'
+import { CardContent, Stack, Typography } from '@mui/material'
+import { useLightCone } from '../Hook'
+
+type LightConeCardProps = {
+  lightConeId: string
+}
+export function LightConeCard({ lightConeId }: LightConeCardProps) {
+  const databaseLightCone = useLightCone(lightConeId)
+  const lightCone = databaseLightCone
+  return (
```

Comment:
Since there is no point displaying the lightcone card if there is no lightcone, you can return early.
```suggestion
  if(!lightCone) return null
  return (
```